src/page/QuizPage.tsx

```diff
@@ -1,11 +1,12 @@
 import { QuizEditor } from '../components/QuizEditor';
 import { QuizVisualization } from '../components/QuizVisualization';
-
+import { Quiz } from '@/components/Quiz';
```

Comment:
```suggestion
import { Quiz } from '@/components/Quiz';

```
p2) 이 개행 관련 내용도 반복적으로 달고 있는데 코드를 읽는 사람이 더 편하게 읽을 수 있는 코드를 쓰는 것도 중요합니다.